commands/kubernetes_test.go

```diff
@@ -337,6 +338,7 @@ func TestKubernetesList(t *testing.T) {
 
 func TestKubernetesCreate(t *testing.T) {
 	testNodePool := testNodePool
```

Comment:
What's the point of `testNodePool := testNodePool`?

Reply:
It was already there when I touched the code :) I think it's meant to make a copy of `testNodePool` to work with that does not interfere with other tests. Although this only has limited effect since `testNodePool` includes slices and maps which are always copied by reference.

Probably something that can be addressed in a future PR (presumably by creating a function that Returns a fresh, new `testNodePool` struct on each invocation).

Reply:
Yeah, I was just curious. But yeah, I guess that makes sense. Thank you for the clarification!

Reply:
I added https://github.com/mitchellh/copystructure as a dependency in a recent PR that's been merged already so you could use that to do a deep copy if you wanted to!